CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ## 🛑 Breaking changes 🛑
 
 - Change configmapprovider.Provider to accept a location for retrieve (#4657)
+- Change Properties Provider to be a Converter (#4666)
```

Comment:
Should we have snippet on how to replace to configmap.NewProperties usage to the new converter in the PR description or here?

Reply:
Not sure what needs to be documented, since this is not directly used, usually used via the `service.NewDefaultConfigProvider` which does the right thing.